src/style.css

```
@@ -1 +1,79 @@
 /* Styles go here */
+* {
+  margin: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.logo {
+  margin: auto auto auto 50px;
+}
+
+.site-header {
+  display: flex;
+  align-items: center;
+  text-transform: uppercase;
+  font-size: 12px;
+  flex-flow: row;
+  padding-right: 50px;
+  font-weight: 500;
+  height: 60px;
+ 
+}
+
+.site-nav {
+  display: flex;
+  float: right;
+}
+
+ul {
+  display: table;
+  padding-inline-start: 0;
+}
+
+li:not(:first-child) {
+  margin-left: 20px;
+}
+
+.site-nav > ul > li {
+  display: inline;
+  padding-bottom: 20px;
+  position: relative;
+}
+
+li a {
+  text-decoration: none;
+  display: inline-block;
+  color: #000;
+  padding: 22.5px 0;
+}
+
+.is-active a {
+  color: #00acdc;
+}
+
+li:after {
```

```suggestion
is-active:after {
```
Тобі потрібно додати псевдоелемент конкретно до одного пункта меню